Frends.PowerShell.RunCommand/Frends.PowerShell.RunCommand/PowerShell.cs

```diff
@@ -34,9 +36,81 @@ public static PowerShellResult RunCommand(RunCommandInput input, [Browsable(fals
     {
         return DoAndHandleSession(options?.Session, (session) =>
         {
-            return ExecuteCommand(input.Command, input.Parameters, input.LogInformationStream, session.PowerShell);
+            if (input.ExecuteNativeShell)
+            {
+                var tempScript = Path.Combine(Path.GetTempPath(), $"{Path.GetRandomFileName()}.ps1");
+                try
+                {
+                    string parameterString = string.Empty;
+                    if (input.Parameters != null)
+                    {
+                        foreach (var param in input.Parameters)
+                        {
+                            parameterString += $"-{param.Name} \"{param.Value}\" ";
+                        }
+                    }
+                    File.WriteAllText(tempScript, $"{input.Command} {parameterString}", Encoding.UTF8);
+                    return ExecuteProcess(tempScript, input.Parameters);
+                }
+                finally
+                {
+                    File.Delete(tempScript);
+                }
+            }
+            else
+                return ExecuteCommand(input.Command, input.Parameters, input.LogInformationStream, session.PowerShell);
         });
     }
+    private static PowerShellResult ExecuteProcess(string scriptPath, PowerShellParameter[] parameters)
+    {
+        List<dynamic> results = new();
+        List<string> errors = new();
+
+
+
+        using Process process = new();
+        process.StartInfo = new()
+        {
+            FileName = "powershell.exe",
```

Comment:
I // think // that on Linux this is not going to work. I have a feeling it is something like `pwsh` there or something like that. So maybe add a check for that and use a different executable name there. Also it would be nice to add to docs that those are the executable names we expect on windows/linux.